package/src/skia/web/JsiSkPath.ts

```diff
@@ -332,38 +344,41 @@ export class JsiSkPath extends HostObject<Path, "Path"> implements SkPath {
     for (let i = 0; i < cmd1.length; i++) {
       if (cmd1[i][0] !== cmd2[i][0]) {
         return false;
+      } else if (cmd1[i][0] === PathVerb.Conic && cmd1[i][5] !== cmd2[i][5]) {
```

Comment:
conic weights need to be identical; this came up when adding the tests from Skia. Now the confidence in the polyfill is quite high